internal/agent-api/client.go

```diff
@@ -23,7 +23,16 @@ type EventCallback func(string, cloudevents.Event)
 type LogCallback func(string, LogEntry)
 
 const (
-	nexTriggerSubject = "x-nex-trigger-subject"
+	NexTriggerSubject = "x-nex-trigger-subject"
+	NexRuntimeNs      = "x-nex-runtime-ns"
+
+	HttpURLHeader = "x-http-url"
```

Comment:
We should('ve) probably had nex in all of these headers, just to make it harder for accidental collisions. For example `x-nex-http-url` instead of `x-http-url`